syntax = "proto3";

service DatabaseCreator {
  rpc create (CreateDatabaseRequest) returns (CreateDatabaseReply) {}
}

message CreateDatabaseRequest {
  string dbName = 1;
}

message CreateDatabaseReply {
  bool wasCreatedSuccessfully = 1;
}